### PR TITLE
Cleaned and remade feature extraction.

### PR DESCRIPTION
**Pull request by MBTMBTMBT**

### CODE REVIEW

File: tasks/receptionist/launch/setup.launch [hunk 1]
```diff
@@ -16,5 +16,10 @@
 
     <!--      PERCEPTION     -->
     <include file="$(find lasr_vision_yolov8)/launch/service.launch" />
+    
+    <node pkg="lasr_vision_yolov8" type="service" name="yolo_service" output="screen"/>
```
Comment (jws-1) [inline tasks/receptionist/launch/setup.launch:20]:
```suggestion
```

File: tasks/receptionist/src/receptionist/head_direction_test.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import rospy
+import tf2_ros
+import geometry_msgs.msg
+
```
Comment (jws-1) [inline tasks/receptionist/src/receptionist/head_direction_test.py:4]: Remove

File: tasks/receptionist/src/receptionist/states/speakdescriptions.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import smach
+import rospy
+
```
Comment (jws-1) [inline tasks/receptionist/src/receptionist/states/speakdescriptions.py:3]: Remove

**Review by jws-1 — CHANGES_REQUESTED** [review]
Awaiting changes

File: common/vision/lasr_vision_bodypix/src/lasr_vision_bodypix/bodypix.py [hunk 4]
```diff
@@ -25,9 +29,15 @@ def load_model_cached(dataset: str) -> None:
         model = loaded_models[dataset]
     else:
         if dataset == 'resnet50':
-            model = load_model(download_model(BodyPixModelPaths.RESNET50_FLOAT_STRIDE_16))
+#             name = download_model(BodyPixModelPaths.RESNET50_FLOAT_STRIDE_16)
+#             rospy.logwarn(name)  /home/bentengma/keras_model/tf-bodypix/tfjs-models-savedmodel-bodypix-resnet50-float-model-stride16
+#             model = load_model(name)
+            model = load_model(path.join(r.get_path("lasr_vision_bodypix"), "models", "keras_model", "tf-bodypix", "tfjs-models-savedmodel-bodypix-resnet50-float-model-stride16"))
+            # model = load_model(download_model(BodyPixModelPaths.RESNET50_FLOAT_STRIDE_16))
```
[the remark this answers is not in the document]
Comment (MBTMBTMBT) [reply common/vision/lasr_vision_bodypix/src/lasr_vision_bodypix/bodypix.py]: Options of loading different models are kept, no longer loading any model from local folder (unless downloaded automatically).

**Review by jws-1 — APPROVED** [review]
Couple of minor comments, but the rest looks good! Looking forward to seeing a live demonstration, and then we can merge this in! :partying_face:

File: common/vision/lasr_vision_feature_extraction/package.xml [hunk 5]
```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0"?>
 <package format="2">
-  <name>lasr_vision_torch</name>
+  <name>lasr_vision_feature_extraction</name>
   <version>0.0.0</version>
-  <description>Serivce providing custom vision models using PyTorch</description>
+  <description>Custormer attribute extraction, encluding face and cloth, implemented with PyTorch customed models.</description>
```
[the remark this answers is not in the document]
Comment (MBTMBTMBT) [reply common/vision/lasr_vision_feature_extraction/package.xml]: Haha sorry! Bad languagev:)

File: skills/src/lasr_skills/vision/get_image.py [hunk 6]
```diff
@@ -27,3 +26,55 @@ def execute(self, userdata):
...
+    def execute(self, userdata):
+        try:
+            for _ in range(5):  # This is a little change, see if helps avoid having delayed frame issue.
+                userdata.img_msg_2d = rospy.wait_for_message(
+                    self.topic1, Image)
+                userdata.img_msg_3d = rospy.wait_for_message(
+                    self.topic2, PointCloud2)
+        except Exception as e:
+            print(e)
+            return 'failed'
```
[the remark this answers is not in the document]
Comment (MBTMBTMBT) [reply skills/src/lasr_skills/vision/get_image.py]: I renamed it and canceled the loop. I think I'll solve this synchronization in the future :)

**Review by jws-1 — APPROVED** [review]
Good job. (Also saw working demo).

**Pull request merged**